packages/blocks/slack/test/wait-for-interaction.test.ts

```diff
@@ -0,0 +1,514 @@
+const slackUpdateMessageMock = jest.fn();
+
+function normalizeEmojiStringMock(str: string): string {
```

Comment:
Why are you mocking it instead of testing the real underlying normalizeEmojiString function?